contrib/IBM/sarama.v1/sarama.go

```diff
@@ -74,6 +74,10 @@ func WrapPartitionConsumer(pc sarama.PartitionConsumer, opts ...Option) sarama.P
 			// kafka supports headers, so try to extract a span context
 			carrier := NewConsumerMessageCarrier(msg)
 			if spanctx, err := tracer.Extract(carrier); err == nil {
+				if linksCtx, err := spanctx.(ddtrace.SpanContextWithLinks); err && linksCtx.SpanLinks() != nil {
```

Comment:
`linksCtx, err := spanctx.(ddtrace.SpanContextWithLinks); err`
Isn't this supposed to be
`linksCtx, err := spanctx.(ddtrace.SpanContextWithLinks); err == nil`
?

Reply:
oops yes that is correct 👍